package.json

```diff
@@ -11,7 +11,7 @@
     "release": "pnpm -F=fastify-renderer publish",
     "preinstall": "npx only-allow pnpm",
     "prerelease": "pnpm -F=fastify-renderer run gitpkg publish",
-    "test": "jest --forceExit -w=1",
```

Comment:
VSCode's debugger insists upon adding `--runInBand` which causes a conflict between the two. They are equivalent flags btw.